3rdParty/boost/CMakeLists.txt

```diff
@@ -125,6 +125,20 @@ target_link_libraries(boost_program_options PUBLIC boost_boost) #this will add r
 target_compile_definitions(boost_program_options PRIVATE "BOOST_SYSTEM_STATIC_LINK=1")
 target_compile_definitions(boost_program_options PUBLIC  "BOOST_ALL_NO_LIB=1")
 
+#build boost_container - add more source files as needed
+add_library(boost_container STATIC
+    ${boost_src}/libs/container/src/alloc_lib.c
+    ${boost_src}/libs/container/src/dlmalloc.cpp
+    ${boost_src}/libs/container/src/global_resource.cpp
+    ${boost_src}/libs/container/src/monotonic_buffer_resource.cpp
+    ${boost_src}/libs/container/src/pool_resource.cpp
+    ${boost_src}/libs/container/src/synchronized_pool_resource.cpp
+    ${boost_src}/libs/container/src/unsynchronized_pool_resource.cpp
+)
+target_link_libraries(boost_container PUBLIC boost_boost) #this will add required includes
+target_compile_definitions(boost_container PRIVATE "BOOST_SYSTEM_STATIC_LINK=1")
```

Comment:
it's definitely not boost system :)
Please add it correct way.
At the same time looks like now you're using only single simple header